### ISSUE TITLE
Select function not working with beta 3.13

### ISSUE THREAD

**Issue by apinturaud** [slotThe/RemarkableLamyEraser#23]
Hello
Since using beta 3.13 the "select" function activate the "select eraser" tool instead.
That may be because of a shifting "index" since the addition of a secondary writing tool in the toolbar since the 3.13 !
Can you deal with the offset due to the increase of the number o tools displayed is the toolbar ? 
I add a screenshot for convenience.
You can see the highlighter as secondary writing tool.
Thank you.
Best regards 

![20240707_153418](https://github.com/slotThe/RemarkableLamyEraser/assets/174946168/f9b2f497-fc9b-439a-b519-cfabfa0960e5)

**Comment by slotThe** [comment]
Indeed, a new tool there would mess up the index that we're accessing. I will take a look at this as soon as I have access to 3.13

**Comment by drvdijk** [comment]
I'm on the `3.13.1.2` stable version now, and indeed I had to change the config to make it somewhat workable again:

```
click
double-click   	       	    redo
triple-click
quadruple-click
quintuple-click
long-click

press&hold                  eraser-erase
double-press&hold
triple-press&hold
quadruple-press&hold
quintuple-press&hold
assume-toolbar-open
```

This makes the double click not redo, but actually undo. I couldn't find a similar trick to make the long-click do a one-off-select to fake the one-off-eraser ;)

**Comment by daeh** [comment]
Confirming that on `3.13.1.2` few things works as intended (indexing is definitely part of it, but might not be the full story).

**Comment by gduprey** [comment]
I'm seeing this on 3.14.1.9 which installed for me today too.  And agree - they seemed to have "moved stuff around"

One thought - to help with the "next time" they move things around - maybe in addition to being able to use names like "erase-selection" and such, we could also just use numbers like "3-2" in place of the words?

I don't see this making the fix easier this time, but if they move things around again, it would give people who receive the update and get stuck before this has a chance to be updated a way to "fix" it with numbers, even if sort of clunky.  They would be able to keep going while this plugin gets to work vs being stuck (and yeah, going forward, I have automatic updates turned off and will check here, but..... )

Just a thought.  Thanks for this plugin - it was the #1 reason I was "stuck" on RMOS 2.x till recently.

Gerry

**Comment by mhantsch** [comment]
Same in 3.16.2.3. The coordinates in effects_data.h (LOCATION_LOOKUP_TOUCH) are no longer correct, as the second writing tool button pushes everything further down.

**Comment by slotThe** [comment]
As mentioned in #28, responsiveness has really taken a hit with recent versions. People are free to try out bf6499e63214934c50be8e227140d7482e5caea3 (right-handed only for now), but unless the toolbar is open, the success rate is not great. Almost makes me wonder whether the arcane sequence of events that must be sent for a touch press to register has changed

**Comment by slotThe** [comment]
FWIW, I just updated to 3.20, and responsiveness seems to have improved quite drastically! All functions now seem to work as expected again, just like in earlier releases.
